tests/test_core.py

```diff
@@ -69,3 +72,50 @@ def resource(self):
         setup_test_table(self.table_name, self.sqlalchemy_url)
         yield
         teardown_test_table(self.table_name, self.sqlalchemy_url)
+
+
+def test_jsonb():
```

Comment:
The way other tests are running needs a refactor I think. Created https://github.com/MeltanoLabs/tap-postgres/issues/114 

Whatever we pick for the standard we should just use it on all of them.